### PR TITLE
add jac_cell unit test

### PR DESCRIPTION
**Pull request by zingale**
this simply displays the analytic and numerical Jacobian elements for comparison

### CODE REVIEW

File: unit_test/jac_cell/jac_cell.H [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#include <fstream>
+#include <iostream>
+#include <iomanip>
+#include <react_util.H>
+#include <numerical_jacobian.H>
+
+using namespace unit_test_rp;
+
+AMREX_INLINE
+void burn_cell_c()
```
Comment (maxpkatz) [inline unit_test/jac_cell/jac_cell.H:17]: Could update this function name.

**Comment by maxpkatz** [comment]
Perhaps add a CI test for this?

File: unit_test/jac_cell/jac_cell.H (hunk 1 above)
Comment (zingale) [reply unit_test/jac_cell/jac_cell.H]: done

**Comment by zingale** [comment]
CI added

**Review by zhichen3 — APPROVED** [review]

**Pull request merged**